README.MD

```diff
@@ -99,7 +99,7 @@ This is an example of how to list things you need to use the software and how to
 
 1. Clone the repo 
 ```sh
-git clone https://github.com/ps2-tyra/tyra.git
```

Comment:
This should not be there.
Probably this is a part from commit from tyra:master?